packages/expo-build-properties/src/android.ts

```diff
@@ -53,10 +59,53 @@ export const withAndroidBuildProperties = createBuildGradlePropsConfigPlugin<Plu
       propName: 'android.enableProguardInReleaseBuilds',
       propValueGetter: (config) => config.android?.enableProguardInReleaseBuilds?.toString(),
     },
+    {
+      propName: 'android.flipper',
+      propValueGetter: (config) =>
+        typeof config.android?.flipper === 'string' ? config.android.flipper : undefined,
+    },
   ],
   'withAndroidBuildProperties'
 );
 
+export const withAndroidFlipper: ConfigPlugin<PluginConfigType> = (config, props) => {
+  const ANDROID_FLIPPER_KEY = 'FLIPPER_VERSION';
+  const FLIPPER_FALLBACK = '0.125.0';
+
+  // when not set, or set to enabled, make no changes
+  if (props.android?.flipper === undefined || props.android.flipper === true) {
+    return config;
```

Comment:
I don't know. Honestly Flipper is enabled by default on Android and has been since RN 0.69. My understanding is that it isn't on-by-default in iOS because of the podfile pain when `use_frameworks` is also present.

I think following the default RN setup (on by default for android, off by default for ios) makes the most sense even if it feels awkward.